docs/machine-learning/resources/tasks.md

```diff
@@ -230,3 +229,27 @@ The forecasting task use past time-series data to make predictions about future
 You can train a forecasting model with the following algorithm:
 
 <xref:Microsoft.ML.TimeSeriesCatalog.ForecastBySsa%2A>
+
+## Image Classification
+
+A [supervised machine learning](glossary.md#supervised-machine-learning) task that is used to predict the class (category) of an image. The input is a set of labeled examples. Each label normally starts as text. It is then run through the TermTransform, which converts it to the Key (numeric) type. The output of the image classification algorithm is a classifier, which you can use to predict the class of new images. Examples of image classification scenarios include:
```

Comment:
Maybe make a note that Image Classification is an multiclass classification algorithm